test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -326,6 +330,45 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
         expect(iamRoleLambdaResource.Properties.RoleName).to.be.eq(customRoleName);
       });
 
+      it('should support `provider.iam.role.path`', async () => {
+        const customRolePath = '/custom-role-path/';
+        const { cfTemplate, awsNaming } = await runServerless({
```

Comment:
Let's reuse `runServerless` configuration setup in _before_ to test this (those calls are expensive and unless it cannot be tested other way, we try to test all customizations with one run)

Reply:
Thanks for the heads up on that! I will make that change tonight.

Reply:
Hi @medikoo - I updated the existing role name test and my new role path test to re-use the existing `runServerless` config from the test setup in that block. 

I did however leave one of the separate usages of it in place though for the invalid path rejection test: that checks a thrown exception, so I believe would need to be called separately as I have it currently.

Please let me know if this works for you. Thank you!